adaptors/hybrisproximityadaptor/hybrisproximityadaptor.cpp

```diff
@@ -96,7 +95,7 @@ void HybrisProximityAdaptor::sendInitialData()
                ProximityData *d = buffer->nextSlot();
                d->timestamp_ = Utils::getTimeStamp();
                d->withinProximity_ = near;
-               d->value_ = near ? 10 : 0;
+               d->value_ = absinfo.value;
```

Comment:
As far as giving a value for failure, normally/ideally proximity sensors return an actual reflectance value. In the Qt proximity plugin, only the value here is used to determine whether it is near or far, so it needs to be a non zero number.